### DISCUSSION TITLE
Curve Optimizer & Per-Core Curve Optimizer

### DISCUSSION THREAD

**Discussion by ghost** [JamesCJ60/Universal-x86-Tuning-Utility#199] [Q&A]
Hello,

Stupid question, but one I have trouble finding an answer.

Are the Per-Core Curve Optimizer values cumulative? Are they added to the All Core values?

e.g. If my all-core curve Optimizer is -15, and i set -5 for Core 1, is Core 1 running at -5 or -20?

Thanks!

Cassio

**Comment by Edensystem** [comment]
Yes they are
